Update with devtech 2026 demo's (#12)
Esri/storymaps-developer-samples: conference-demos/Developer Summit/2026/Readme.md

## Demo instructions

> Demo's for the 2026 Esri Developer and Tech summit.

Reviewer: ```suggestion
Demos from the 2026 Esri Developer and Technology Summit.
```

Plural does not use an apostrophe. Should use full name of conference with correct capitalization.

> All demo's use the python API version 2.4.3

Reviewer: ```suggestion
All demos use the python API version 2.4.3
```
